driver/helpers.c

```diff
@@ -0,0 +1,50 @@
+#include "helpers.h"
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+
+#ifdef _WIN32
+int setenv(const char *name, const char *value, const int overwrite) {
+    if (overwrite != 0 && getenv(name) != NULL) return -1;
+    const size_t n = strlen(name) + 1 /* = */ + strlen(value) + 1 /* \0 */;
+    char *env = calloc(n, sizeof(char));
+    if (env == NULL) return -1;
+    snprintf(env, n, "%s=%s", name, value);
+    const int errcode = _putenv(env);
+    free(env);
+    return errcode;
+}
+#endif
+
+const char *getenv_or_default(const char *name, const char *default_value) {
+    const char *value = getenv(name);
+    if (value == NULL) return default_value;
+    return value;
+}
+
+bool getenv_bool(const char *name, const bool default_value) {
```

Comment:
I suggest it named `getenv_bool_or_default` to inform this function with a default value.